core/store/migrations/migration1600881493/migration.go

```diff
@@ -0,0 +1,22 @@
+package migration1600881493
+
+import "github.com/jinzhu/gorm"
+
+const up = `
+ALTER TABLE keys
```

Comment:
WARNING: BPTXM is going to use the is_funding key! You need to add an exception for it in there.

Reply:
Good point! I made the necessary changes - I think!